### PR TITLE
38280 Improving JavaScript code for performance

### PR DESCRIPTION
**Pull request by yeneastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/573

changed log :
Minify javascript & remove unused code

### CODE REVIEW

**Comment by andernath** [comment]
@yeneastgate 
Could you merge master in this branch. Maybe there are some conflicts, I'm not sure.

**Comment by yeneastgate** [comment]
> @yeneastgate Could you merge master in this branch. Maybe there are some conflicts, I'm not sure.

Yes, I will check and merge it.

**Comment by yeneastgate** [comment]
@andernath I fixed the conflict, please help us to review. Thanks!

File: js/ajax_settings.js [hunk 1]
```diff
@@ -62,7 +61,6 @@ onOffice.ajaxSaver = function(outerDiv) {
 			var message = onOffice.settings[responseMessageKey];
 			var getUrl = window.location.href;
 
-			var getUrlPageEdit = getUrl.split( '&' );
```
Comment (andernath) [inline js/ajax_settings.js:65]: 👍

File: js/onoffice-reference-estate-select.js [hunk 2]
```diff
@@ -1,11 +1,9 @@
-jQuery(document).ready(function () {
+jQuery(document).ready(function ($) {
     const HIDE_REFERENCE_ESTATE = '0';
     const SHOW_REFERENCE_ESTATE = '1';
     const SHOW_ONLY_REFERENCE_ESTATE = '2';
-    const mainElement = document.querySelector('select[name=oopluginlistviews-listtype');
```
Comment (andernath) [inline js/onoffice-reference-estate-select.js:5]: 👍

**Review by andernath — APPROVED** [review]
I will start testing

**Comment by andernath** [comment]
@yeneastgate I finished testing. Can I merge to develop or do you need something to change before or after merge?
I saw your comment here: https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/573#issuecomment-1681744422

**Comment by yeneastgate** [comment]
> @yeneastgate I finished testing. Can I merge to develop or do you need something to change before or after merge? I saw your comment here: [#573 (comment)](https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/573#issuecomment-1681744422)

@andernath I made changes in previous commits! You can merge it
Note: I will update the src in branch #600 after you merge this branch to master

**Pull request merged**